src/States/MainMenuState.cpp

```diff
@@ -211,6 +215,8 @@ void MainMenuState::onFadeComplete()
 void MainMenuState::btnNewGameClicked()
 {
 	if (mFileIoDialog.visible()) { return; }
+	if(dlgOptions.visible()) { return; }
```

Comment:
Minor point, but the spacing between some of the `if` and `(` is different. The code base is a little inconsistent here in that it contains both:
- `if (` 980 occurrences
- `if(` 38 occurrences

I think I would prefer `if (`, and that matches the majority of the code.

----

As an aside, if you're doing a rebase anyway, this might be a fun exercise to figure out how to patch up a previous commit. :wink:

Reply:
Rebasing is evil. I'm just making things worse. :P

Reply:
Hah, I see that. Maybe try again. Did you specify `master` as the base to rebase on top of?

Reply:
I'm planning on making a fix warnings pass next. I'll see if I can't get most of them in that.